modules/programs/firefox/mkFirefoxModule.nix

```diff
@@ -61,7 +61,8 @@ let
     }) // {
       General = {
         StartWithLastProfile = 1;
-        Version = 2;
+      } // lib.optionalAttrs (cfg.profileVersion == null) {
```

Comment:
this should be `cfg.profileVersion != null`, didn't notice that at first review.